AnalysisTools/python/templates/MuonBaseFlow.py

```diff
@@ -13,7 +13,7 @@ def makeAnalysisStep(self, stepName, **inputs):
         if stepName == 'preselection':
             self.addGhostCleaning(step)
             step.addBasicSelector('m', 'pt > 5 && (isGlobalMuon || isTrackerMuon)')
-
+            self.addMuonPOGIDs(step)
```

Comment:
Embedding the IDs should happen in the `preliminary` step if you need them for your preselections, or the `embedding` step if you don't. The `preselection`step should just be for minimal initial selections.